cubi_isa_templates/isatab-stem_cell_core_sc/{{cookiecutter.__output_dir}}/a_{{cookiecutter.assay_prefix}}_{{cookiecutter.assay_name}}.txt

```diff
@@ -1,15 +1,15 @@
-{% set sample_names = cookiecutter.sample_names.split(",") -%}
+{% set source_names = cookiecutter.source_names.split(",") -%}
```

Comment:
This actually is the sample name and not the source name

Reply:
It's the source_name variable being re-used to define sample names. Before it was the sample_name variable being re-used to define source names.

Reply:
You're right. Reverted this change.